Source: Jorge Aparicio
Upstream: https://code.google.com/p/lz4/issues/detail?id=91
Reason: lz4c (needed for kernel compression) is built but not installed

Index: Makefile
===================================================================
--- Makefile	(revision 106)
+++ Makefile	(working copy)
@@ -93,10 +94,12 @@
 install: lz4
 	@install -d -m 755 $(BINDIR)/ $(MANDIR)/
 	@install -m 755 lz4 $(BINDIR)/lz4
+	@install -m 755 lz4 $(BINDIR)/lz4c
 	@install -m 644 lz4.1 $(MANDIR)/lz4.1
 
 uninstall:
 	[ -x $(BINDIR)/lz4 ] && rm -f $(BINDIR)/lz4
+	[ -x $(BINDIR)/lz4c ] && rm -f $(BINDIR)/lz4c
 	[ -f $(MANDIR)/lz4.1 ] && rm -f $(MANDIR)/lz4.1
 
 dist: clean
